### DISCUSSION TITLE
Continuous loading icon on notifications page until manually refreshed

### DISCUSSION THREAD

**Discussion by ghost** [community/community#65538] [Mobile]
When opening the notifications page in the latest update, it continuously shows a loading icon until the page is pulled down to refresh.

https://github.com/community/community/assets/95468948/3d86a0f9-32b6-4263-89f6-193ebec69928

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.128.0 (108490092)
Device: iPhone 13 Pro (iOS 17.0)
</details>

**Comment by trixieboyle** [comment]
> When opening the notifications page in the latest update, it continuously shows a loading icon until the page is pulled down to refresh.
> 
> https://github.com/community/community/assets/95468948/3d86a0f9-32b6-4263-89f6-193ebec69928
> 
> <details>
> <summary>ℹ️ Additional Debug Info</summary>
> GitHub for iOS, Version 1.128.0 (108490092)
> Device: iPhone 13 Pro (iOS 17.0)
> </details>
